tawazi/node/node.py

```diff
@@ -53,22 +54,22 @@ def count_occurences(id_: str, exec_nodes: Dict[str, "ExecNode"]) -> int:
         exec_nodes (Dict[str, ExecNode]): the dictionary of ExecNodes
 
     Returns:
-        int: the number of occurences of id_ in exec_nodes
+        int: the number of occurrences of id_ in exec_nodes
     """
     # example: id_ = "a"
     # ExecNode a is called five times, hence we should have ids a, a<<1>>, a<<2>>, a<<3>>, a<<4>>
```

Comment:
either get rid of those comments or include them directly in the fn docstring I'd say

Reply:
ok